ansible/roles/real-postfix/files/postfix/pass-scripts.mit.edu-suffix

```diff
@@ -0,0 +1,3 @@
+# Match quentin+foo@scripts.mit.edu and rewrite to quentin@+foo
+# This allows the chained LDAP lookup to resolve quentin.scripts.mit.edu with %u
+/([^+]+)([^@]*)@scripts\.mit\.edu/ $1@$2
```

Comment:
Should this be `[^+@]` instead of `[^+]`, and anchored? Also, and more importantly, doesn't this translate `quentin+foo@scripts.mit.edu` to `quentin+@foo`, not `quentin@+foo`?

Reply:
Oh, no, that `+` is a regex metacharacter, not a literal. This would also match `quentin@scripts.mit.edu` and translate it to `quentin@`, at least if the lookup got far enough, right? If that's true and it works (and it kinda feels like it should? unless `%D` is an error with an empty domain part), it might be nice to consolidate the two `pipemap`s into just one.

Reply:
I'm not sure if it's an error for %D to be empty... I'd rather not change it since I know this works. As you noted, the + is handled correctly.